test/api-digester/Outputs/stability-stdlib-abi.without.asserts.swift.expected

```diff
@@ -0,0 +1,133 @@
+Func Double.acos(_:) has been removed
```

Comment:
This looks as if we have the digester baseline set to a recent master here. That seems wrong to me -- @nkcsgexi, shouldn't we use the 5.0 (or 5.1) state?

Reply:
Yup, I agree (was in the process of writing up a radar about that =))

Reply:
<rdar://problem/54653088>